lib/cdo/firehose.rb

```diff
@@ -18,6 +17,9 @@
 #       data_json: "{\"key\":\"value\"}"    # OPTIONAL
 #     }
 #   )
+
+STREAM_NAME = 'analysis-events'
```

Comment:
nit: Use `freeze` on constants.

Reply:
Thanks for continuing to help us improve our code hygiene, @ashercodeorg !  I also saw a big PR from you adding `.freeze` to a bunch of places. What is the long term vision here? Ruby 2.3 supports the `frozen_string_literal` comment and command line flag, and also Ruby 3.0 will freeze string literals by default.

Reply:
My long term vision is to reenable the rubocop `Style/MutableConstant` cop. Though freezing strings may have limited value, freezing other data types has definite value (has caught bugs).